internal/usecase/interactor/user.go

```diff
@@ -166,8 +166,8 @@ func (i *User) Signup(ctx context.Context, inp interfaces.SignupParam) (u *user.
 
 func (i *User) GetUserByCredentials(ctx context.Context, inp interfaces.GetUserByCredentials) (u *user.User, err error) {
 	u, err = i.userRepo.FindByNameOrEmail(ctx, inp.Email)
-	if err != nil && !errors.Is(err, rerror.ErrNotFound) {
-		return nil, err
+	if err != nil || u == nil {
+		return nil, interfaces.ErrInvalidUserCredentials
```

Comment:
I did not say errors from the DB should be erased. Internal errors returned by the handler are automatically output to the standard output, so it is necessary to return the internal errors of the DB in order to determine the cause.

```suggestion
	if err != nil && !errors.Is(rerror.ErrNotFound, err) {
		return nil, err
	} else if u == nil {
		return nil, interfaces.ErrInvalidUserCredentials
	}
```